### PR TITLE
Shubham Feature To Be Merge TO Develop edited

### PR DESCRIPTION
**Pull request by manishmowall**
first pull request in shubham edited

### CODE REVIEW

**Comment by manishmowall** [comment]
yoooo yooo

**Comment by nitishm-webonise** [comment]
second comment

**Comment by manishmowall** [comment]
third comment edited

**Comment by manishmowall** [comment]
fourth comment edited edited edited

**Comment by manishmowall** [comment]
5th comment

**Review by manishmowall — COMMENTED** [review]
look great

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea
```
Comment (manishmowall) [inline .gitignore:1]: ok

**Review by manishmowall — COMMENTED** [review]
ok review edited

File: .gitignore (hunk 1 above)
Comment (manishmowall) [reply .gitignore:1]: second ok

Comment (manishmowall) [reply .gitignore:1]: add second review ok

**Review by manishmowall — COMMENTED** [review]
ok second

**Pull request merged**